analysed-packages/imx-gpu-viv/version-6.4.3.p4.6/README.md

```diff
@@ -0,0 +1,17 @@
+## Download Location
+
+https://github.com/Freescale/kernel-module-imx-gpu-viv/archive/refs/tags/upstream/6.4.3.p4.6.zip
+
+## Package URL (purl)
+
+pkg:generic/imx-gpu-viv@6.4.3.p4.6?download_url=https://github.com/Freescale/kernel-module-imx-gpu-viv/archive/refs/tags/upstream/6.4.3.p4.6.zip
+
+## Creator
+
+Sascha Dierberg
+
+## Reviewers
```

Comment:
Depending on the real package please provide if it is dual licensed (like the one I downloaded) the following comment

## Comment

This package is dual licensed under GPL-2.0-or-later OR MIT. To facilitate automatic use, a default license choice of MIT is denoted in the Acknowledgment section of the OSS disclosure document. This shall not restrict the freedom of users to choose either GPL-2.0-or-later OR MIT. Anyway, all license texts are provided.